test/e2e/config/docker.yaml

```diff
@@ -135,7 +135,7 @@ intervals:
   default/wait-machine-pool-nodes: ["5m", "10s"]
   default/wait-delete-cluster: ["3m", "10s"]
   default/wait-machine-upgrade: ["20m", "10s"]
-  default/wait-machine-pool-upgrade: ["5m", "10s"]
+  default/wait-machine-pool-upgrade: ["20m", "10s"]
```

Comment:
Locally this is much faster on my machine, do we really need to x4 the current timeout?

Reply:
we don't, I was just testing to see if the timeout was the issue. I'll revert it once I get it working.